app/src/test/java/org/dhis2/utils/DateUtilsTest.java

```diff
@@ -68,303 +69,303 @@ public void expiryPeriodAndDaysInRange() throws ParseException {
-        assertEquals("2017-01-01", DateUtils.uiDateFormat().format(minDateYearly2));
-        assertEquals("2018-05-01", DateUtils.uiDateFormat().format(minDateFinancialApril));
-        assertEquals("2017-05-01", DateUtils.uiDateFormat().format(minDateFinancialApril2));
-        assertEquals("2018-07-01", DateUtils.uiDateFormat().format(minDateFinancialJuly));
-        assertEquals("2017-07-01", DateUtils.uiDateFormat().format(minDateFinancialJuly2));
-        assertEquals("2017-10-01", DateUtils.uiDateFormat().format(minDateFinancialOct));
+        assertEquals("2018-07-23", DateUtils.oldUiDateFormat().format(minDateWeekly));
+        assertEquals("2018-07-30", DateUtils.oldUiDateFormat().format(minDateWeekly2));
+        assertEquals("2018-07-29", DateUtils.oldUiDateFormat().format(minDateDaily));
+        assertEquals("2018-07-25", DateUtils.oldUiDateFormat().format(minDateWeeklyWednesday));
+        assertEquals("2018-07-26", DateUtils.oldUiDateFormat().format(minDateWeeklyThursday));
+        assertEquals("2018-07-28", DateUtils.oldUiDateFormat().format(minDateWeeklySaturday));
+        assertEquals("2018-07-29", DateUtils.oldUiDateFormat().format(minDateWeeklySunday));
+        assertEquals("2018-07-16", DateUtils.oldUiDateFormat().format(minDateBiWeekly));
+        assertEquals("2018-06-18", DateUtils.oldUiDateFormat().format(minDateBiWeekly2));
+        assertEquals("2018-07-01", DateUtils.oldUiDateFormat().format(minDateMonthly));
+        assertEquals("2018-06-01", DateUtils.oldUiDateFormat().format(minDateMonthly2));
+        assertEquals("2018-07-01", DateUtils.oldUiDateFormat().format(minDateBiMonthly));
+        assertEquals("2018-05-01", DateUtils.oldUiDateFormat().format(minDateBiMonthly2));
+        assertEquals("2018-05-01", DateUtils.oldUiDateFormat().format(minDateQuarterly));
+        assertEquals("2018-01-01", DateUtils.oldUiDateFormat().format(minDateQuarterly2));
+        assertEquals("2018-07-01", DateUtils.oldUiDateFormat().format(minDateSixMonthly));
+        assertEquals("2017-07-01", DateUtils.oldUiDateFormat().format(minDateSixMonthly2));
+        assertEquals("2018-04-01", DateUtils.oldUiDateFormat().format(minDateSixMonthlyApril));
+        assertEquals("2017-10-01", DateUtils.oldUiDateFormat().format(minDateSixMonthlyApril2));
+        assertEquals("2018-01-01", DateUtils.oldUiDateFormat().format(minDateYearly));
+        assertEquals("2017-01-01", DateUtils.oldUiDateFormat().format(minDateYearly2));
+        assertEquals("2018-05-01", DateUtils.oldUiDateFormat().format(minDateFinancialApril));
+        assertEquals("2017-05-01", DateUtils.oldUiDateFormat().format(minDateFinancialApril2));
+        assertEquals("2018-07-01", DateUtils.oldUiDateFormat().format(minDateFinancialJuly));
+        assertEquals("2017-07-01", DateUtils.oldUiDateFormat().format(minDateFinancialJuly2));
+        assertEquals("2017-10-01", DateUtils.oldUiDateFormat().format(minDateFinancialOct));
```

Comment:
why asserting the oldUIDateFormat, and not the new one?

Reply:
just checking the result